### DISCUSSION TITLE
Need Help in Local Development of Outline in MacOS

### DISCUSSION THREAD

**Discussion by ANUSHMANGAL01** [outline/outline#7361] [Self Hosting]
I have been facing problems (for days now) in setting up Outline in my local environment on MacOS.
I follow this guide : https://docs.getoutline.com/s/hosting/doc/local-development-5hEhFRXow7 step by step.

'make up' command runs fine and the backend server seems to be running. However, the frontend is not running and I see a blank white screen on going to https://localhost:3000/

I feel there is some problem in frontend running. Some of the text that I see on terminal after running 'make up' , is as follows:

![Image 07-08-24 at 3 50 PM](https://github.com/user-attachments/assets/97a3cb8c-df3d-45a7-a78c-3ea52b9cd716)

I think the frontend is crashing as it says '[vite] page reload build/server/error.prod.html' 
I don't know the real reason.

The backend seems to be running fine. Correct me if I am wrong.
![Image 07-08-24 at 3 52 PM](https://github.com/user-attachments/assets/c8e3db07-cb5e-4711-8669-4aef46fb8a7c)

On opening 'https://localhost:3000/' on Chrome, I see a blank white screen. On inspecting, I see this:
![Image 07-08-24 at 3 53 PM](https://github.com/user-attachments/assets/326f71b7-fc36-43ce-800c-66f4438e66c4)

Sometimes it shows net::ERR_CERT_AUTHORITY_INVALID, sometime CONNECTION_REFUSED, and sometimes a CORS error.

This is what I see on Firefox:

![Image 07-08-24 at 4 05 PM](https://github.com/user-attachments/assets/025ac1bd-935e-477d-85cc-120fc44e662e)

Please help me with this... I am stuck at this for days now.

**Comment by quickstar** [comment]
I have exactly the same issue. @tommoor any ideas what we might doing wrong?

**Comment by foxxyz** [comment]
Are you sure you want to set an SSL cert in development? I've been able to run Outline successfully locally on MacOS with the following `.env` settings:

```
NODE_ENV=development
URL=http://localhost:3000
DATABASE_URL=postgres://user:pass@127.0.0.1:5432/outline
PGSSLMODE=disable
REDIS_URL=redis://127.0.0.1:6379
FORCE_HTTPS=false
```

**Reply by tommoor** [reply]
You can set the URL to `https:\\local.outline.dev` - it's a domain we keep for this purpose and should work well.

SSL certs are auto provisioned in dev through make up
